test/pylib/rest_client.py

```diff
@@ -213,6 +213,9 @@ async def get_enabled_injections(self, node_ip: str) -> list[str]:
         assert(type(e) == str for e in data)
         return data
 
+    async def message_injection(self, node_ip: str, injection: str) -> None:
```

Comment:
I assume you tested this manually for now (a real test we can add later when we have all the pieces)

Reply:
Yes, exactly as we agreed to do.